metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -771,6 +786,7 @@ class MetalsLanguageServer(
             scalaVersionSelector
           )
         )
+        buildTargets.addData(ammonite.buildTargetsData)
```

Comment:
Same question here, could we add ammonite data when we make the connection to ammonite?

Reply:
These can be wired upfront, so there's no reason not to do it IMO.